ci-operator/step-registry/ovn/conf/internal-masquerade-subnet/ovn-conf-internal-masquerade-subnet-commands.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cat >> "${SHARED_DIR}/manifest_cluster-network-03-config.yml" << EOF
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  defaultNetwork:
+    ovnKubernetesConfig:
+      gatewayConfig:
+        ipv4:
+          internalMasqueradeSubnet: 100.254.169.0/29
+EOF
+
+cat ${SHARED_DIR}/manifest_cluster-network-03-config.yml
```

Comment:
Out of curiosity, are all manifests in `${SHARED_DIR}` among all the other files in there, picked up for installation? or is there any more specific criteria?

Reply:
great question. I don't actually know the answer, just following the pattern that was there since the beginning for me :)